test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -38,7 +38,7 @@ def hook(context):
     DEFAULT_CONFIG = {
         'localBackendUrl': 'https://localhost:9200/',
         'secureLocalBackendUrl': 'https://localhost:9200/',
-        'maxSyncTimeout': 10,
+        'maxSyncTimeout': 60,
```

Comment:
I understand that we need to increase the timeout. But, must it be 60? cc @saw-jan